extensions/python/PythonCreator.h

```diff
@@ -83,6 +83,7 @@ class PythonCreator : public minifi::core::CoreComponent {
 
  private:
   void registerScriptDescription(const std::string& class_name, const std::string& full_name, const std::filesystem::path& path, const std::string& script_name) {
+    GlobalInterpreterLock gil_lock;
```

Comment:
what does this lock ensure?

Reply:
good catch, this seems unneccesary. Removed it in https://github.com/apache/nifi-minifi-cpp/pull/1504/commits/3053dd3f1972734aa54e4d679dbc59a64e79352e